op5build/libexec/op5_scheduled_reports.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+import argparse
+import signal
+import subprocess
+import threading
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--interval", type=int, default=60)
+    args = parser.parse_args()
+    _run(args.interval)
+
+
+def _run(interval):
+    event_exit = threading.Event()
+
+    def _handler(*_):
+        event_exit.set()  # unblocks wait()
+
+    signal.signal(signal.SIGTERM, _handler)
+    signal.signal(signal.SIGINT, _handler)
+
+    while not event_exit.is_set():
+        # Run the subprocess and let it fail loudly on non-zero exit code.
```

Comment:
Is it expected that this will often fail, and if so why do we want it to fail loudly? Just curious.

Reply:
I don't think it's expected. But _if_ it would fail, I'd like it to show up in systemd journal to aid users/CS to troubleshoot. This way, this process will crash with `CalledProcessError` exception and print stuff to stderr which journal picks up. Then systemd restarts us.

Reply:
Very reasonable.